src/main/java/com/example/compound/repositories/UserRepositoryImpl.java

```diff
@@ -72,7 +72,8 @@ public Integer getCountByEmail(String email) {
     @Override
     public User findById(int id) {
         System.out.println("I'm here" + id);
-        return jdbcTemplate.queryForObject(SQL_GET_USER_BY_UUID, new Object[]{id}, userRowMapper);
+        // changed from: return jdbcTemplate.queryForObject(SQL_GET_USER_BY_UUID, new Object[]{id}, userRowMapper);
+        return jdbcTemplate.queryForObject(SQL_GET_USER_BY_UUID, userRowMapper, id); // TODO: Is this right?
```

Comment:
Is this right?

Reply:
We need to check if it works. If it does then we can make the change.